templates/docs/patterns/hero/index.md

```diff
@@ -15,64 +15,102 @@ Depending on the size and composition of your content, you can choose from a var
 
 The hero pattern is composed of the following elements:
 
-| Element              | Description                                                              |
-| -------------------- | ------------------------------------------------------------------------ |
-| Title (**required**) | Title text (to be placed in `h1` heading)                                |
-| Subtitle             | Subtitle text (using `h2` style)                                         |
-| Description          | Description text (one or more paragraphs)                                |
-| Call to action block | [Call to action block](/docs/patterns/cta-block) beneath the description |
-| Image                | Main hero visual                                                         |
+| Element              | Description                                                   |
+| -------------------- | ------------------------------------------------------------- |
+| Title (**required**) | Title text (to be placed in `h1` heading)                     |
+| Subtitle             | Subtitle text (using `h2` style)                              |
+| Description          | Description text (one or more paragraphs)                     |
+| Call to action block | [CTA block](/docs/patterns/cta-block) beneath the description |
+| Image                | Main hero visual                                              |
 
 ## Jinja Macro
 
 ### Parameters
 
 <div style="overflow: auto;">
-  <table style="overflow: visible; width: auto;">
```

Comment:
You've removed auto width from the hero params table, but used it for tiered list params table. It should probably be consistent between the two of them.

Without the auto width, I'm seeing a very scrunched table on mobile

![image](https://github.com/user-attachments/assets/826906d0-aed8-432d-ad98-bc41852f045d)

The tiered list params table looks good on mobile; it horizontally scrolls ![image](https://github.com/user-attachments/assets/7203e7e8-7977-4c42-ab51-85831bd57e55)

Reply:
I'm fine with this (this is how I originally tried to implement this table). However @bartaz suggested to me to use percentages for the column widths, we should see what he thinks.